### PR TITLE
Change misleading wording

### PR DESCRIPTION
**Pull request by dotcode**
Previous wording gave the impression we have achieved full compliance. Although in many areas we achieve very substantial compliance (far above the norms for the industry), we must not give the impression of full compliance where that does not exist.

### CODE REVIEW

**Comment by dotcode** [comment]
Using privileges to squash and merge without review. Deal with it.

File: accessibility/house-style.md [hunk 1]
```diff
@@ -59,8 +59,7 @@ You may use [our VPAT template](https://github.com/springernature/vpat) when eva
 
 ## How we conform
 
-### Standards we conform to
-
+### Standards we aim for
 Our target standards are [WCAG 2.1](https://www.w3.org/TR/WCAG21/) level AA.
```
Comment (josebolos) [inline accessibility/house-style.md]: Needs an empty line between the title and the paragraph so markdownlint doesn't complain about it 😬

Comment (dotcode) [reply accessibility/house-style.md]: Typical 🤦‍♂️

Comment (dotcode) [reply accessibility/house-style.md]: On the upside, happily proving my own maxim that "there is no change too small for a review".

**Pull request merged**